sormas-backend/src/main/java/de/symeda/sormas/backend/task/TaskFacadeEjb.java

```diff
@@ -440,6 +444,11 @@ public long count(TaskCriteria taskCriteria) {
 			filter = CriteriaBuilderHelper.and(cb, filter, criteriaFilter);
 		}
 
+		Predicate contactPredicate = this.contactRightsPredicate(cb ,task, joins.getAssignee(), joins.getTaskObservers());
+		if(contactPredicate != null){
+			filter = CriteriaBuilderHelper.and(cb, filter, contactPredicate);
+		}
```

Comment:
please move the contactPredicate under createUserFilter. This way you will also cover the exports and the Android app synchronization.